test/suites/container_oci.sh

```diff
@@ -0,0 +1,19 @@
+test_container_oci() {
+  if [ -n "${INCUS_OFFLINE:-}" ]; then
+    echo "==> SKIP: Skipping OCI tests as running offline"
+    return
+  fi
+
+  ensure_has_localhost_remote "${INCUS_ADDR}"
+  incus network create inct$$
+
+  incus remote add docker https://docker.io --protocol=oci
+  incus launch docker:hello-world --console --ephemeral --network=inct$$
```

Comment:
I don't exactly know the semantics of `--network=...` here, but I wonder if this is something that should be implied for OCI containers?

Reply:
It's just something that's needed for the testsuite.

Normal users will have an `incusbr0` bridge that's part of their default profile, providing them with an `eth0` that has both IPv4 and IPv6 connectivity.

Our testsuite instead normally just has a basic `p2p` device for tests, providing no connectivity. Because OCI containers run a DHCP client on startup, I had to make the test use a temporary network that does provide DHCP so the test would run quickly and not hit the 5s DHCP timeout for every container.

Reply:
I see, makes sense, thanks.